src/App.jsx

```diff
@@ -1,11 +1,17 @@
-import './App.css';
-import {Errbound} from './components/errBoundary';
-
+import "./App.css";
+import Errbound from "./components/errBoundary/errBoundary";
```

Comment:
```suggestion
import Errorboundary from "./components/ErrorBoundary/ErrorBoundary";
```

try to use meaningul and readable names

Reply:
All suggested changes with updated code have been pushed to the errBoundary branch